infrastructure/helm/quantumserverless/Chart.yaml

```diff
@@ -17,3 +17,5 @@ dependencies:
     condition: rayClusterEnable
   - name: manager
     condition: managerEnable
+  - name: redis
+    condition: redisEnable
```

Comment:
Not something to fix in this PR but something that I just come to my mind reading again your messsage, @IceKhan13 . Does it make sense that the user  can disable `ray`? Being this the core of the deployment.

I suppose that the user can only deploy the jupyter to have the examples and the notebook running. It can connect it later on with its external instance but I don't know. Just curious about your opinion here.